pnpm-lock.yaml

```diff
@@ -18829,6 +18841,18 @@ packages:
       source-map: 0.6.1
     transitivePeerDependencies:
       - supports-color
+    dev: true
+
+  /istanbul-lib-source-maps@5.0.4:
```

Comment:
Did you replace it everywhere? The latest version is not removed

Reply:
The remaining v4's are `dev: true` only. We have dependency on `jest` which brings in the older version.